src/parsec_security/parsec_threat_model/threat_model.md

```diff
@@ -320,6 +326,35 @@ cleared.
 | I | The attacker can read all the confidential assets placed in local memory by the service. | M-8        | AS1, AS2, AS4 and AS5 |
 
+### Attacker "SPIFFE Validation Request" - A12
+
+This attacker uses the SPIFFE Workload API endpoint created by the SPIFFE implementation to
+interfere with the validation of the JWT-SVID token sent by the client.
+
+*Only applicable for deployment (JWTS).*
+
+|   | Description                                                                                                      | Mitigation                                                                                                                                                                                                           | Assets   |
+|---|------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| I | An attacker steals the authentication token from a client's request to then execute any operation in their name. | M-9                                                                                                                                                                                                                  | AS4, AS3 |
+| D | An attacker modifies the validation request to make it fail.                                                     | M-9                                                                                                                                                                                                                  | AS6      |
+| D | An attacker overloads/crashes the SPIFFE implementation and thus prevents the service from making use of it.     | SPIFFE implementation specific. An ["Unavailable" error](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE_Workload_Endpoint.md#6-error-codes) can be used for implementations to perform load-shedding. | AS6      |
+
+### Attacker "SPIFFE Validation Response" - A13
+
+This attacker uses the SPIFFE Workload API endpoint created by the SPIFFE implementation to
+interfere with the validation of the JWT-SVID token sent by the client. It can also create a spoofed
+endpoint, mimicking the SPIFFE implementation's one.
+
+*Only applicable for deployment (JWTS).*
+
+|   | Description                                                                                                                | Mitigation | Assets                  |
+|---|----------------------------------------------------------------------------------------------------------------------------|------------|-------------------------|
+| S | An attacker impersonates the SPIFFE implementation: Parsec sends the authentication data of clients to malicious entity.   | O-14       | AS1                     |
```

Comment:
Isn't what you describe here more for A12? Would've said that spoofing for the response means the attacker sending their own response to the service. 

Tbh, I'm not sure for communication channels like this or between client and service if having separate attackers for each direction makes that much sense 😕

Reply:
hmm for me A12 is when the attacker either mimick the Parsec service or spy on the request and A13 is when the attacker mimicks the SPIFFE implementation or spy on the response.

Under this definition I did not add a lot of threats for A12 because then it would be doing the SPIFFE implementation threat model.

I agree though that it might not make sense to split this into two and is a bit messy... Maybe in the future we want to merge our attackers. For this one I can either merge now or leave it like that?

Reply:
Yeah, I think it makes sense to merge them in the future, I can't understand why we created attackers that are mostly in other project's threat model.